packages/backend-api/src/api/services/updateNotifications.ts

```diff
@@ -170,15 +183,14 @@ async function getArticleUpdate(articleId: number) {
   if (!article) {
     return null;
   }
-  const aData: any = pick(article.toJSON(), articleFields);
-  aData.assignedModerators = aData.assignedModerators.map((i: any) => i.moderator_assignment.userId.toString());
+  const aData = serialiseObject(article, ARTICLE_FIELDS);
 
   const category = await Category.findById(
-    aData.categoryId,
+    await article.get('categoryId'),
     {include: [{ model: User, as: 'assignedModerators', attributes: ['id']}]},
   );
-  const cData: any = pick(category.toJSON(), categoryFields);
-  cData.assignedModerators = cData.assignedModerators.map((i: any) => i.user_category_assignment.userId.toString());
+
+  const cData: any = serialiseObject(category, CATEGORY_FIELDS);
```

Comment:
I think here `cdata` is a `string[]` is that right? 
 
Adding types makes the code much easier to understand, so if we can, lets try and use complete types. :)

Reply:
You are right.  I'll fix.